packages/components/src/ui/elevation/stories/index.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import { number } from '@storybook/addon-knobs';
+/**
+ * Internal dependencies
+ */
+import { Elevation } from '../index';
+import { View } from '../../view';
+
+export default {
+	component: Elevation,
+	title: 'G2 Components (Experimental)/Elevation',
+};
+
+export const _default = () => {
+	const value = number( 'value', 5 );
+	const hover = number( 'hover', undefined );
+	const focus = number( 'focus', undefined );
+	const active = number( 'active', undefined );
+
+	return (
+		<View
+			css={ {
+				padding: '20vh',
+				position: 'relative',
+				margin: '20vh auto 10vw',
+				maxWidth: 400,
+			} }
+		>
+			<Elevation
+				isInteractive
+				value={ value }
+				hover={ hover }
+				focus={ focus }
+				active={ active }
+			/>
+		</View>
```

Comment:
Thinking more about this, I think this should be considered a bad use, right? Since the direct parent element (`View`) can't be focused, the `focus` prop makes no sense?

Reply:
That is true. I've updated the story to remove `focus` from the `_default` story.